README.md

```diff
@@ -1,29 +1,72 @@
-# Chillzone
-
-> Will find you a chillzone inside IIT KGP at any time
-
-## TOC
-
-- [Organization of the code](#organization-of-the-code)
-- [How does it work?](#how-does-it-work)
+<div id="top"></div>
+
+<div align="center">
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+<!-- [![MIT License][license-shield]][license-url] -->
+[![Wiki][wiki-shield]][wiki-url]
+
+</div>
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/metakgp/chillzone">
+    <img width="140" alt="image" src="images/icons/icon_384x384.png">
+  </a>
+
+  <h1 align="center">Chillzone</h1>
+
+  <p align="center">
+    <i>Will find you a chillzone inside IIT KGP at any time</i>
+    <br />
+    <a href="https://github.com/metakgp/chillzone/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/metakgp/chillzone/issues">Request Feature</a>
```

Comment:
Can replace one of these with website link and other one can be "Report Bug / Request Feature" or "Give Feedback"

Reply:
What should the text be for website?

Reply:
Just "Website" or "Chillzone" or something. Do you have any suggestions?

Reply:
Webiste is okay:
![image](https://github.com/metakgp/chillzone/assets/139053348/02f42594-6c21-4b1b-941a-3cd50bf4f9e2)

Or pura link hi daal sakte hai:
![image](https://github.com/metakgp/chillzone/assets/139053348/4113050b-907b-428a-99c5-b383544edee3)

Reply:
website hi achha hai